contracts/alice/src/state.rs

```diff
@@ -16,21 +16,18 @@ impl Alice {
         // no-op
     }
 
-    pub fn withdraw(self: &mut State<Self>, wfct: Wfct) {
-        let _: bool = self
-            .transact(rusk_abi::transfer_module(), "wfct", &wfct)
+    pub fn withdraw(&mut self, wfct: Wfct) {
+        let _: bool = rusk_abi::call(TRANSFER_CONTRACT, "wfct", &wfct)
             .expect("Transparent withdrawal transaction should succeed");
     }
 
-    pub fn withdraw_obfuscated(self: &mut State<Self>, wfco: Wfco) {
-        let _: bool = self
-            .transact(rusk_abi::transfer_module(), "wfco", &wfco)
+    pub fn withdraw_obfuscated(&mut self, wfco: Wfco) {
+        let _: bool = rusk_abi::call(TRANSFER_CONTRACT, "wfco", &wfco)
             .expect("Obfuscated withdrawal transaction should succeed");
     }
 
-    pub fn withdraw_to_contract(self: &mut State<Self>, wfctc: Wfctc) {
-        let _: bool = self
-            .transact(rusk_abi::transfer_module(), "wfctc", &wfctc)
+    pub fn withdraw_to_contract(&mut self, wfctc: Wfctc) {
+        let _: bool = rusk_abi::call(TRANSFER_CONTRACT, "wfctc", &wfctc)
             .expect("Obfuscated withdrawal transaction should succeed");
```

Comment:
Change the panic message to not be equal to `withdraw_obfuscated` one

Reply:
:+1: